vtrace/platforms/posix.py

```diff
@@ -61,6 +61,14 @@ def handleAttach(self):
         # break after our library load events to make things easy
         self.runAgain(False)  # Clear this, if they want BREAK to run, it will
         self.fireNotifiers(vtrace.NOTIFY_BREAK)
+        # POSIX hack - Windows signals on library load, POSIX doesn't
+        self._LibraryLoadHack()
+        
+    def _LibraryLoadHack(self):
+        '''
+        Implement at the platform level
+        '''
+        pass
```

Comment:
PTRACE doesn't provide events for LOAD_LIBRARY, so here we add a per-platform call on `_LibraryLoadHack()`.  currently, linux is implemented.